### PR TITLE
Materials overhaul

### PR DESCRIPTION
**Pull request by bombasticSlacks**
Materials have been simplified extremely, 

Differentiation between reagents and materials has been removed. 

Materials come in 
* simple
* artisan
* exotic

Types of Materials are:
* Metal
* Leather
* Fabric
* Wood
* Flora
* Resonant
* Bits
* Liquid

These can be used for different craft types interchangeably within tier.

Martial grade items have all been renamed to reference artisan instead. 

Spell book has been removed from the game,
Reagent Bag now is required for any ritual and has charges, it's now the primary casting tool.

### CODE REVIEW

File: .obsidian/app.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "Duelist",
     "Cryptozoologist",
     "Herbologist",
-    "vampiric"
+    "vampiric",
```
Comment (KoojKooj) [inline .obsidian/app.json:10]: Not sure if important so bringing it up, capitalization consistency?

File: .obsidian/app.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
...
+    "icor"
```
Comment (KoojKooj) [inline .obsidian/app.json:11]: Do you mean "ichor"?

File: Game/Materials.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
+## Material
+All things are made up of materials. This usually isn't a major concern to the average adventurer, but a savvy tradesman will want to carefully select materials of varying qualities for their projects. Characters will need to source materials for anything they are [Crafting](Crafting). Materials should be simple and interchangeable withing a quality.
```
Comment (KoojKooj) [inline Game/Materials.md:2]: Third last word, withing -> within

File: Game/Materials.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+### Material Quality
+All materials generally come in three quality levels:
+
+#### Basic
+simple materials, not very useful for crafting anything but the most basic of things.
```
Comment (KoojKooj) [inline Game/Materials.md:8]: Missed capital "S"

File: Game/Materials.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+#### Artisan
+Well treated, high-quality materials used for proper tools and equipment.
+#### Exotic
+Impressive materials, sourced from unique locations and situations with powerful properties.
+
+### Material Types
+There are generally 8 types of materials that will be used for creating:
+
+#### Bits
+After killing a creature you can, unless it’s been annihilated, usually [harvest](Equipment#Looting) some of what are called bits. Bits are a generalization of organs, glands, bones, and other valuables. Your character would intuitively know what parts of a creature are valuable, even if you don’t yet.
```
Comment (KoojKooj) [inline Game/Materials.md:18]: Readability change:  "[...] usually harvest some of what are called bits." -> "[...] usually harvest some bits" || "[...] usually harvest bits"

File: Game/Materials.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Basic Bits
+Simple unimpressive creature viscera, not useful for much more than cooking or filler.
+
+> Blood, mundane organs, bones
+
+##### Artisan Bits
+Purposeful materials from creatures with clearly developed biological systems doing useful things.
+
+> An undead creature's icor, the claws of a powerful predator, eyes of a creature that can see in the dark
```
Comment (KoojKooj) [inline Game/Materials.md:28]: "ichor," probably spell-check related

File: Game/Materials.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Basic Flora
+Simple plants useful for herbal remedies and basic purposes.
+
+> Wild flowers, tobaco
+
+##### Artisan Flora
+Specialized plants with great properties possible to derive.
+
+> Psychadelic mushrooms, poppeys, powerful medicinal herbs
```
Comment (KoojKooj) [inline Game/Materials.md:48]: "poppey" -> "poppy"

File: Game/Materials.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Exotic Flora
+Plants with truley magical properties.
```
Comment (KoojKooj) [inline Game/Materials.md:51]: "truley" -> "truly"

File: Game/Materials.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+
+> Junk metal melted down, pig iron, poorly sourced metal.
+
+##### Artisan Metals
+Simple unimpressive metals that are built for purpose.
+
+> Refined iron, copper, brass.
+
+##### Exotic Metals
+Complex impressive metals with properties great for working with.
```
Comment (KoojKooj) [inline Game/Materials.md:71]: Readability suggestion:  "Complex impressive metals with properties great for working with." -> "Complex, impressive metals with great working properties."

File: Game/Materials.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+
+> Steel, mithril, dwarven ingots.
+
+---
+
+#### Leather
+Repurposed skin of creature's, leather makes an excellent material for [Storage](Storage), and [Armour](Armour).
```
Comment (KoojKooj) [inline Game/Materials.md:78]: "creature's" -> "creatures"

File: Game/Materials.md [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Basic Leather
+Simple animal leathers.
+
+> cow leather, deer hide.
```
Comment (KoojKooj) [inline Game/Materials.md:83]: Caps "Cow"

File: Game/Materials.md [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Artisan Leather
+Hardened leathers.
+
+> boiled leather, treated with epoxy, other treatments.
```
Comment (KoojKooj) [inline Game/Materials.md:88]: Caps "Boiled"

File: Game/Materials.md [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Artisan Leather
+Hardened leathers.
+
+> boiled leather, treated with epoxy, other treatments. 
+
+##### Exotic Leather
+Exotic creature skins with stranger properties.
+
+> Dragon hide, demons skin.
```
Comment (KoojKooj) [inline Game/Materials.md:93]: "demons skin" -> "demon's skin"
Suggestion:  make both examples possessive or neither, i.e., "Dragon's hide, demon's hide" OR "Dragon hide, demon skin"

File: Game/Materials.md [hunk 14]
```diff
@@ -0,0 +1,168 @@
...
+
+##### Exotic Resonant
+Intentionally imbued powders, items warped from long exposure to enchanted places.
+
+> Crystallized magic, liquified elements
+
+---
+
+#### Liquid
+Spirits, elixirs, strange fluids, anything that can be used to dissolve material, treat it or as lubrication. Useful in crafting [Comestibles](Comestibles).
```
Comment (KoojKooj) [inline Game/Materials.md:159]: Readability suggestion: "Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials."

File: Game/Materials.md (hunk 9 above)
Comment (bombasticSlacks) [reply Game/Materials.md:71]:
```suggestion
Complex, impressive metals with great working properties.
```

File: Game/Materials.md (hunk 10 above)
Comment (bombasticSlacks) [reply Game/Materials.md:78]:
```suggestion
Repurposed skin of creatures, leather makes an excellent material for [Storage](Storage), and [Armour](Armour).
```

File: Game/Materials.md (hunk 11 above)
Comment (bombasticSlacks) [reply Game/Materials.md:83]:
```suggestion
> Cow leather, deer hide.
```

File: Game/Materials.md (hunk 12 above)
Comment (bombasticSlacks) [reply Game/Materials.md:88]:
```suggestion
> Boiled leather, treated with epoxy, other treatments. 
```

File: Game/Materials.md (hunk 13 above)
Comment (bombasticSlacks) [reply Game/Materials.md:93]:
```suggestion
> Dragon hide, demon skin.
```

File: Game/Materials.md (hunk 14 above)
Comment (bombasticSlacks) [reply Game/Materials.md:159]:
```suggestion
Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials. Useful in crafting [Comestibles](Comestibles).
```

**Pull request merged**